examples/playground/json-examples/3d-heatmap.json

```diff
@@ -1,4 +1,5 @@
 {
+  "google": true,
```

Comment:
We don't have a google token for building our examples yet so maybe drop this change in this PR?